examples/cpp/helloworld/CMakeLists.txt

```diff
@@ -55,6 +55,23 @@ if(GRPC_AS_SUBMODULE)
   set(_PROTOBUF_PROTOC $<TARGET_FILE:protoc>)
   set(_GRPC_GRPCPP_UNSECURE grpc++_unsecure)
   set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:grpc_cpp_plugin>)
+elseif(GRPC_FETCHCONTENT)
+  # Another way is to use CMake's FetchContent module to clone gRPC at
+  # configure time. This makes gRPC's source code available to your project,
+  # similar to a git submodule.
+  include(FetchContent)
+  FetchContent_Declare(
+    grpc
+    GIT_REPOSITORY https://github.com/grpc/grpc.git
+    GIT_TAG        v1.25.0)

```

Comment:
when used in a distribtest, setting the GIT_TAG makes no sense. Nor does master. The distribtest should test with whatever gRPC commit is being under test (e.g. for PRs, it must test the version from that PR - otherwise it's useless).

Reply:
👍 

I came up with what I think is a "best of both worlds" approach. The example still specifies v1.25.0, but we override it in the distribtest by setting `FETCHCONTENT_SOURCE_DIR_GRPC` to the existing clone of GRPC.

Reply:
The override is fair, but I think it makes the test fragile. Also, the tag number will quickly go out of date.
How about setting GIT_TAG to a non-existent tag:
```
# when using gRPC, you will actually set this to an existing tag, such as v1.25.0, v1.26.0 etc..
# for the purpose of testing, we override the tag used to the commit that's currently under test.
GIT_TAG        vGRPC_TAG_VERSION_OF_YOUR_CHOICE
```